### ISSUE TITLE
Your opinion/advice is badly needed

### ISSUE THREAD

**Issue by eugene-panferov** [ninenines/erlang.mk#708]
I happened to be affected by the rebar3 vs erlang.mk controversy, in an unpleasant practical sense.
I foolishly bought into the "industry standard" mantra and end up with a malfunctioning release.
Therefore I want to hear your side of the "debate" before I move back to erlang.mk -- if you already have all your points stated, then, please, give me a link.

erlang.mk worked for me so well that i have never bothered to learn how does it work, it simply makes a release that runs.

rebar3 has made a release launching script that does not launch epmd and run_erl (I assume i did something wrong, but without your help i have failed to find any solution so far)

please, reassure me/us that erlang.mk will survive despite "industry standard"

**Comment by essen** [comment]
Erlang.mk is here to stay.

Erlang.mk is not going anywhere, it works very well, including for very large projects spanning 30+ applications like RabbitMQ. The remaining few bugs are slowly getting fixed, and functionality keeps being added or improved.

OTP team was supposed to move many applications out of the OTP 20 release, they managed to remove one. Turns out doing the things they need done with rebar3 is very difficult. Makefiles are really good at getting things built, so Erlang.mk is truly necessary.

Even if for any reasons I had to stop working on Erlang.mk, the project would still be maintained by developers at RabbitMQ or other. In fact most of the recent improvements have come from third parties, not from me.

Note however that as far as releases are concerned, we are using Relx (same as rebar3), and therefore may be hit by issues there. On the other hand, we control which version of Relx is pulled, and that version is not increased if there are issues in the current Relx, so we've been able to avoid creating problems for people this way. For example the last time it was increased it it took a few weeks because of an issue that I ended up sending a patch to Relx for.

Of course people can also help financially if they want to make sure Erlang.mk sticks around. :-) I would love to be able to spend a few days to fix the remaining issues. But there's no cause for alarm. It works really well, is very low maintenance, and it doesn't look like that's going to change.
